### DISCUSSION TITLE
Support different colors for `striped` prop for `Table`

### DISCUSSION THREAD

**Discussion by mcmxcdev** [mantinedev/mantine#3112] [Feature requests]
`Table` component currently only supports gray stripes, but we have a business requirement to have the table stripes in our companies brand color. It would be great to support that!

![Screenshot from 2022-12-05 09-02-08](https://user-images.githubusercontent.com/16797721/205584709-fe50454f-5e56-4077-8b07-592619a7ec10.png)

**Comment by rtivital** [comment]
You can add styles to Table component with `sx` prop
